src/python/devcontainer-feature.json

```diff
@@ -22,6 +22,11 @@
       "default": "os-provided",
       "description": "Select a Python version to install."
     },
+    "additionalVersions": {
```

Comment:
We intentionally did not publicize this option just because it was experimental. Also, it was meant only to be used by `universal` image.

However, if it works great in different test scenarios then it is worth publicizing 🎉